src/Modal/ModalPopup.jsx

```diff
@@ -58,13 +87,16 @@ ModalPopup.propTypes = {
   ]),
   /** Specifies position according to the element that the ``positionRef`` prop points to */
   placement: PopperElement.propTypes.placement,
+  /** Caret to the modal window pointing to the button */
```

Comment:
nit: rather than explicitly saying "button" here, let's say "target" as the target does not always have to be a button.

related, let's call "modal window" the "modal popup"

Reply:
Corrected